ca/certificate-authority.go

```diff
@@ -117,6 +135,16 @@ func NewCertificateAuthorityImpl(config cmd.CAConfig, clk clock.Clock, issuerCer
 		return nil, err
 	}
 
+	var hsmFaultTimeout time.Duration
+	if config.HSMFaultTimeout == "" {
+		hsmFaultTimeout = defaultHSMFaultTimeout
+	} else {
+		hsmFaultTimeout, err = time.ParseDuration(config.HSMFaultTimeout)
```

Comment:
You can get this same behavior with a field of type ConfigDuration. It will default to the zero value of Duration, I believe, so you can check for that.

Also, thanks for config gating this change. The ideal way to do it, rather than choosing a default, is to make it so the code isn't exercised when the flag is not set. For instance, in `checkHSMFault` you would always return success if `ca.hsmFaultTimeout == 0`. That way deploying fresh code changes the minimum number of things, and you can roll out individual config changes once the deploy is done.